packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js

```diff
@@ -320,16 +328,16 @@ export default class LocusInfo extends EventsScope {
           },
           EVENTS.DESTROY_MEETING,
           {
+            reason: CALL_REMOVED_REASON.SELF_LEFT,
             shouldLeave: false
           }
         );
       }
     }
     else if (this.parsedLocus.fullState.type === _MEETING_) {
-      if (this.fullState && this.fullState.state === LOCUS.STATE.INACTIVE ||
-         this.fullState.state === LOCUS.STATE.TERMINATING) {
-        // We are destroying the meeting here . There might be instance where we need to
-        // change the state and not remove the meeting
+      if (this.fullState && (this.fullState.state === LOCUS.STATE.INACTIVE || this.fullState.state === LOCUS.STATE.TERMINATING)) {
+        // TODO: update the meeting state
+        // this.updateMeeting({state: 'ENDED'});
```

Comment:
What is the purpose of this commented code?

Reply:
dont think its needed as we. destroy the meeting object anyways